docs/getting-started-guides/kubeadm.md

```diff
@@ -98,7 +98,16 @@ For each host in turn:
 
 * SSH into the machine and become root if you are not already (for example,
   run `sudo su -`).
-* If the machine is running Ubuntu or HypriotOS, run:
+
+* If the machine is running Ubuntu or Debian:
+
+  ``` bash
+  apt install docker.io snapd
+  snap install kubeadm --classic
```

Comment:
This will always target `stable` channel, which moves with the upstream stable 1.6.1, 1.6.2, 1.7.0, etc